src/lib/uri/uri.c

```diff
@@ -10,9 +10,18 @@
 #define XSTRDUP(s)      (s != NULL ? xstrdup(s) : NULL)
 #define XSTRNDUP(s, n)  (s != NULL ? xstrndup(s, n) : NULL)
 
+/**
+ * WARNING: this structure is exposed in Lua via FFI (see src/lua/uri.lua): any
+ * change  must be reflected in `ffi.cdef`.
+ */
```

Comment:
Nit: `WARNING`, `CAUTION`, `READ IT`, `IMPORTANT` and so on are redundant for my taste. One either read comments or doesn't.

Reply:
Feel free to resolve the discussion without any action, it is just about my personal feeling.

Reply:
Okay so, closing.